src/api/test/functional/maintenance_test.rb

```diff
@@ -901,6 +901,10 @@ def test_manual_branch_with_extend_names
   end
 
   def test_create_maintenance_project_and_release_packages
+    # FIXME: https://github.com/rails/rails/issues/37270
+    (ActiveJob::Base.descendants << ActiveJob::Base).each(&:disable_test_adapter)
+    ActiveJob::Base.queue_adapter = :inline
+
```

Comment:
I finally had to use this trick to force `:inline` adapter, as in many other tests. Now the tests pass everywhere.